.babelrc

```diff
@@ -1,6 +1,5 @@
 {
   "presets": [
-    "es2015",
```

Comment:
oh, forgot to mention the main reason to move to this - this upgrade is required to use the javascript syntax used to cleanup and move away from `.createClass` and towards static props and es6 classes.